README.md

```diff
@@ -7,29 +7,42 @@ This repository contains the formalization of Cedar and infrastructure for perfo
 * [`cedar-lean`](./cedar-lean) contains the Lean formalization of, and proofs about, Cedar.
 * [`cedar-drt`](./cedar-drt) contains code for fuzzing, property-based testing, and differential testing of Cedar.
 * [`cedar-policy-generators`](./cedar-policy-generators) contains code for generating schemas, entities, policies, and requests using the [arbitrary](https://docs.rs/arbitrary/latest/arbitrary/index.html#) crate.
-* `cedar` is a git submodule, pinned to the associated commit of [cedar](https://github.com/cedar-policy/cedar).
 
 ## Build
 
-To build the Lean formalization and proofs:
+### Lean formalization and proofs
 
 * Install Lean, following the instructions [here](https://leanprover.github.io/lean4/doc/setup.html).
 * `source cedar-drt/set_env_vars.sh`
 * `cd cedar-lean && lake build Cedar`
 
-To build the DRT framework:
+### DRT framework
+
+The simplest way to build our DRT framework is to use the included Dockerfile:
+
+```bash
+docker build . -t cedar_drt # ~10 minutes
+docker run -it cedar_drt
```

Comment:
Might want to suggest `--rm`, which I think is a very sensible default when using `-it`